### ISSUE TITLE
Add companion contact sensor

### ISSUE THREAD

**Issue by justjam2013** [fyodorvi/homebridge-tuya-laundry-notify#2]
**Describe the solution you'd like:**
Add a companion contact sensor. The sensor can send notifications without an external service, but primarily it can be used as a trigger for HomeKit automations.

**Additional context:**
Currently this plugin just sends notifications when the laundry machine finishes and power consumption drops. However a companion contact sensor can trigger any automation, for example: turning lights on/off, sending a voice notification to HomePod(s)*, or any other automation.

*My HomeKit automations send voice notifications to HomePods by connecting to a local Piper TTS server. Piper generates mp3/wav files and stores them in a media folder shared with the homebridge-homepod-radio plugin. The automation then toggles the plugin to AirPlay the audio file to a HomePod.

**Comment by fyodorvi** [comment]
Wow that’s such a cool idea with audio notifications! I had no idea that was possible.

The plugin does have exposeStateSwitch in the config, have a look at the readme. The switch state reflects the status of a laundry device. Would that work for you?

**Comment by justjam2013** [comment]
Ah, the last entry in the example config! 😁
Might I suggest adding a paragraph in the readme about using that config setting?

Also, Pushed.co configuration is required. I could enter dummy values in the config for the `pushed` properties, but if there is no Pushed account, it would eventually fail in `PushGateway.send()` and pepper the log with `"Failed to send push through Pushed.co: Blah blah blah"` errors.
Let's say that in the future the code was modified to check that the account exists. At that point my automation would stop working for no apparent reason.
If the Pushed account was optional, then notifications would be optional. Maybe make the switch required if there was no account or just mention it in the readme and let the user enable it.

TL;DR: yes, that would work, but it would be exploiting an internal condition in the current code.

Edit: The Home app natively supports push notifications for Sensors, but not for Switches.
So a sensor doubles up as both a trigger and a source for notifications, with no third party service or app required.

**Comment by justjam2013** [comment]
Closing this as I no longer need it.
